src/wasm-interpreter.h

```diff
@@ -160,6 +160,17 @@ struct FuncData {
   }
 };
 
+// The data of a (ref exn) literal.
+struct ExnData {
+  // The tag of this exn data.
+  Tag* tag;
```

Comment:
This probably also needs an instance `self` pointer to differentiate tags defined by different instances of the same module.

Reply:
Good point, I suppose it does. I'll add a TODO (atm we don't fuzz or test with that afaik)

Reply:
testsuite/try_table.wast depends on this (so we skip it atm).

Reply:
Oh, good. Then we can enable that later. (I just prefer to keep this PR small.)